torch/fx/operator_schemas.py

```diff
@@ -265,12 +255,12 @@ def create_type_hint(x):
             if isinstance(x, list):
 
                 def ret_type(x):
-                    return List[x]  # type: ignore[valid-type]
+                    return list[x]  # type: ignore[valid-type]
 
             else:
 
                 def ret_type(x):
-                    return Tuple[x, ...]
+                    return tuple[x, ...]
```

Comment:
Huh, I guess this is ambiguous without type hints, isn't it? I think you would need to use a TypeVarTuple or something here, or at least the classic typing. Not sure this actually worked well before.

Reply:
I'm not sure. It's better to revert changes in this file.